packages/semantic-dom-diff/chai-dom-diff-plugin.d.ts

```diff
@@ -1,19 +1,37 @@
 /// <reference types="chai" />
 
-declare namespace Chai {
-  interface Assertion {
-    dom: Assertion;
-    lightDom: Assertion;
-    shadowDom: Assertion;
-    notEqual(actual: Object, expected: Object, message?: string): void;
-    equalSnapshot(options?: Object): Assertion;
-    notEqualSnapshot(options?: Object): Assertion;
-  }
+import { DiffOptions } from "./get-diffable-html";
+
+declare global {
+  namespace Chai {
+    interface Assertion extends LanguageChains, NumericComparison, TypeComparison {
+      dom: Assertion;
+      lightDom: Assertion;
+      shadowDom: Assertion;
+      notEqual(actual: Object, expected: Object, message?: string): void;
+      equalSnapshot(options?: Object): Assertion;
+      notEqualSnapshot(options?: Object): Assertion;
+    }
+
+    interface Assert {
+      dom: Pick<Assertion, 'equal'|'notEqual'|'equalSnapshot'|'notEqualSnapshot'>;
```

Comment:
I think a "truer" approach would be to implement the entire interface in JavaScript, just throw errors on nonsense assertions. 

cc: @LarsDenBakker

Reply:
I didn't write this code, so I'm not sure :p

Reply:
YOLO